docs/root/configuration/statistics.rst

```diff
@@ -21,8 +21,8 @@ Server related statistics are rooted at *server.* with following statistics:
   :widths: 1, 1, 2
 
   uptime, Gauge, Current server uptime in seconds
-  memory_allocated, Gauge, Current amount of allocated memory in bytes
-  memory_heap_size, Gauge, Current reserved heap size in bytes
+  memory_allocated, Gauge, Current amount of allocated memory in bytes. Total of both new and old Envoy processes on hot restart. 
+  memory_heap_size, Gauge, Current reserved heap size in bytes. Old Envoy process heap size on hot restart. 
```

Comment:
I don't think this is correct. This is the current/new process heap size only.

Reply:
my bad..copy paste issue. sorry. If we had that info, we would have combined both of them :-)

Reply:
@mattklein123 one more thing, many of the statistics with long descriptions are forcing scroll bars in the tables which looks very odd as we have to scroll every time we need to read those stats descriptions. I found a solution by applying css. Is there a restriction on using custom css or should I submit a PR for that?

Reply:
Yes, if you know how to do the CSS please fix. The other thing that really bothers me is how emphasis text is rendered in red. If you can fix that with CSS that would be awesome also.